source/release-notes/3-1-6.rst

```diff
@@ -8,19 +13,25 @@ Crafter CMS 3.1.6
 New Features
 ^^^^^^^^^^^^
 
-* Added workflow / publishing support for S3 bucket storage. 
-* Added support for Dashboard APIs that enables content retrieval based on content and metadata criteria defined by users.
-* Added a new video picker control for mediaconvert data source
+* Added ability to manage assets in external storage through workflow and publishing mechanics. |br|
+  *See* :ref:`publishing-assets-in-external-storage` *for more information.*
+* Added support for Dashboard APIs that enables content retrieval based on content and metadata criteria defined by users. |br|
+  *Read more about it:* https://app.swaggerhub.com/apis/craftercms/studio/#/dashboard
+* Added a new video picker control for mediaconvert data source |br|
+  *See* :ref:`form-transcoded-video` *for more information.*
 * Added support for cache warming configuration per site
```

Comment:
to enable each site to define how it gets its cache warmed up. {where is a link to the config or docs?}